designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview.component.html

```diff
@@ -216,14 +216,22 @@ <h3 class="text-center">
                             data-ng-if="$ctrl.readOnly">
                         {{ experiment.doi }}
                     </strong>
-                    <div style="display:inline-block; width:20%;">License</div>
+                    <div style="display:inline-block; width:20%;">License(s)</div>
```

Comment:
maybe make this "Licenses"
```suggestion
                    <div style="display:inline-block; width:20%;">Licenses</div>
```